en/meta_attributes.rst

```diff
@@ -0,0 +1,98 @@
+
+Translated from source: `STAN_d_DEF_2011-06-22_eCH-0117 Meta-Attribute für INTERLIS-Modelle.pdf 
+<https://www.ech.ch/alfresco/s/ech/download?nodeid=788eb38a-bf2b-4f3d-96a8-addc37bba41f>`_ (German) from eCH www.interlis.ch
+
+Meta Attributes in Interlis Files
+------------------------------------------
+
+**Comment vs. Attribute**
+
+An Interlis comment starts with ``!!`` and ends with a line end. An attribute starts with ``!!`` as well but followed by an ``@``:
+
+**Syntax**
+
+After the start of ``!!@`` the attribute name follows, then equal, then attributevalue:
+
+::
+
+	!!@<name>=<value>
+
+Followed by the referenced element (MODEL, TOPIC, CLASS etc.)
+
+**Example**
+
+::
+
+	!!@ furtherInformation=https://www.astra.admin.ch/
+	MODEL ExceptionalLoadsCatalogues_V1 (en)
+	AT "http://models.geo.admin.ch/ASTRA/"
+	VERSION "2017-02-08"  =
+	  IMPORTS CatalogueObjects_V1,Units;
+	  !!@ topicInformation="Route-Type"
+	  TOPIC TypeOfRouteCatalogue
+	  EXTENDS CatalogueObjects_V1.Catalogues =
+	    !!@ dispExpression="type||' sometext'"
+	    CLASS TypeOfRoute
+	    EXTENDS CatalogueObjects_V1.Catalogues.Item =
+
+``furtherInformation`` is referenced to ``ExceptionalLoadsCatalogues_V1``, ``topicInformation`` to ``TypeOfRouteCatalogue`` and ``dispExpression`` to ``TypeOfRoute``.
+
+For more complex usage see the `specification of the Verein eCH <https://www.ech.ch/alfresco/s/ech/download?nodeid=788eb38a-bf2b-4f3d-96a8-addc37bba41f>`_
+
+Meta Attributes in the Database
+------------------------------------------
+
+When importing the data from the Interlis file to the DB, with ili2db integrated in the Project Generator, the Meta Attributes are stored in the table **t_ili2db_meta_attrs**:
+
+===============================================================  ==================  ===========================
+ilielement                                                       attr_name           attr_value
+===============================================================  ==================  ===========================
+ExceptionalLoadsCatalogues_V1                                    furtherInformation  https://www.astra.admin.ch/
+ExceptionalLoadsCatalogues_V1.TypeOfRouteCatalogue               topicInformation    Route-Type
+ExceptionalLoadsCatalogues_V1.TypeOfRouteCatalogue.TypeOfRoute   dispExpression      type||' sometext'
+===============================================================  ==================  ===========================
+
+Project Generator Specific Attributes
+================================================
+
+Some attributes the Project Generator can handle as properties in the QGIS project.
```

Comment:
Some additional non standard meta attributes are understood by the project generator.